Dell.CommandConfigure v5.1.0

andreasbrett opened the pull request:
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/234337)

andreasbrett committed:
Dell.CommandConfigure v5.1.0

andreasbrett commented:
Verified URL and hash. Not sure why the pipeline has issues downloading the installer. Maybe Dell blocks access from the Azure pipeline servers?

stephengillie commented:
Automatic Validation ended with:
> Error occurred while downloading installer from https://dl.dell.com/FOLDER12758784M/1/Dell-Command-Configure-Application_PNVDY_WIN64_5.1.0.24_A00.EXE. Forbidden (403). (0x80190193).
 Error information: 
 Http Response: OK
 Http Reason Phrase: OK
 
 3/3/2025 12:42:17 PM

Automated error analysis suggests 80190193 may mean Forbidden (403)

(Automated response - build 988.)

stephengillie commented:
URL: https://dl.dell.com/FOLDER12758784M/1/Dell-Command-Configure-Application_PNVDY_WIN64_5.1.0.24_A00.EXE 
Response status code does not indicate success: 403 (Forbidden).

(Automated message - build 988)

stephengillie commented:
Automatic Validation ended with:
> Error occurred while downloading installer from https://dl.dell.com/FOLDER12758784M/1/Dell-Command-Configure-Application_PNVDY_WIN64_5.1.0.24_A00.EXE. Forbidden (403). (0x80190193).
 Error information: 
 Http Response: OK
 Http Reason Phrase: OK
 
 3/3/2025 12:42:17 PM

Automated error analysis suggests 80190193 may mean Forbidden (403)

(Automated response - build 988.)

stephengillie commented:
URL: https://dl.dell.com/FOLDER12758784M/1/Dell-Command-Configure-Application_PNVDY_WIN64_5.1.0.24_A00.EXE 
Response status code does not indicate success: 403 (Forbidden).

(Automated message - build 988)

andreasbrett commented:
Could someone that is not a bot take a look?

andreasbrett committed:
Merge branch 'microsoft:master' into dcc-5.1.0

stephengillie commented:
Automatic Validation ended with:
> Error occurred while downloading installer from https://dl.dell.com/FOLDER12758784M/1/Dell-Command-Configure-Application_PNVDY_WIN64_5.1.0.24_A00.EXE. Forbidden (403). (0x80190193).
 Error information: 
 Http Response: OK
 Http Reason Phrase: OK
 
 3/4/2025 5:59:05 PM

(Automated response - build 990.)

stephengillie commented:
URL: https://dl.dell.com/FOLDER12758784M/1/Dell-Command-Configure-Application_PNVDY_WIN64_5.1.0.24_A00.EXE 
Response status code does not indicate success: 403 (Forbidden).

(Automated message - build 990)

andreasbrett commented:
I don't get error 403 on my side. Maybe your client IP is banned at Dell?

stephengillie commented:
> I don't get error 403 on my side. Maybe your client IP is banned at Dell?

If the publisher have blocked both our data center, and my residential IP address, based on a few requests - then it's possible that the host might be too sensitive to make a good fit for this package manager.

andreasbrett committed:
test other Dell URL

testing if URL of DCC 5.0.1 works in the Azure pipeline (doesn't produce 403 errors)

andreasbrett commented:
> > I don't get error 403 on my side. Maybe your client IP is banned at Dell?
> 
> If the publisher have blocked both our data center, and my residential IP address, based on a few requests - then it's possible that the host might be too sensitive to make a good fit for this package manager.

I agree, then all Dell manifests would need to be removed from winget. I just tested if the URL from [DCC 5.0.1](https://github.com/microsoft/winget-pkgs/pull/210322) is also non-functional and it is. This URL though also works for me every time I try to fetch it. So I don't think it's rate limiting but rather something like user-agent based blocking. Then again you said you tested with your residential IP address and it did also not work.

andreasbrett committed:
reverting to correct DCC 5.1.0 URL

reverting previous test

stephengillie commented:
Automatic Validation ended with:
> Installer Verification Analysis Context Information: 
 Error occurred while downloading installer from https://dl.dell.com/FOLDER12758784M/1/Dell-Command-Configure-Application_PNVDY_WIN64_5.1.0.24_A00.EXE. Forbidden (403). (0x80190193).
 Error information: 
 Http Response: OK
 Http Reason Phrase: OK
 
 3/8/2025 11:02:21 AM
 
 ****************************************************************************************
 Static Analysis
 ****************************************************************************************

(Automated response - build 994.)

stephengillie commented:
URL: https://dl.dell.com/FOLDER12758784M/1/Dell-Command-Configure-Application_PNVDY_WIN64_5.1.0.24_A00.EXE 
Response status code does not indicate success: 403 (Forbidden).

(Automated message - build 994)

ItzLevvie commented:
Close with reason: Stale pull request;
